bumble/smp.py

```diff
@@ -1850,19 +1853,23 @@ def on_smp_pairing_public_key_command(
             elif self.pairing_method == PairingMethod.PASSKEY:
                 self.send_pairing_confirm_command()
         else:
-            if self.pairing_method == PairingMethod.PASSKEY:
-                self.display_or_input_passkey()
 
-            # Send our public key back to the initiator
-            self.send_public_key_command()
+            def next_steps():
```

Comment:
To enforce type checking
```suggestion
            def next_steps() -> None:
```

Reply:
done